charts/linkerd2/README.md

```diff
@@ -0,0 +1,139 @@
+|`ClusterDomain`                       | Kubernetes DNS Domain name to use                                                    | `cluster.local`|
+|`EnableH2Upgrade`                     | Allow proxies to perform transparent HTTP/2 upgrading                                |`true`|
+|`ImagePullPolicy`                     | Docker image pull policy                                                             |`IfNotPresent`|
+|`LinkerdVersion`                      | Control plane version                                                                |`stable-2.5.0`|
+|`Namespace`                           | Control plane namespace                                                              |`linkerd`|
+|`OmitWebhookSideEffects`              | Omit the sideEffects flag in the webhook manifests                                   |`false`|
+|`WebhookFailurePolicy`                | Failure policy for the proxy injector                                                |`Ignore`|
+|`ControllerImage`                     | Docker image for the controller, tap and identity components                         |`gcr.io/linkerd-io/controller`|
+|`ControllerLogLevel`                  | Log level for the control plane components                                           |`info`|
+|`ControllerReplicas`                  | Number of replicas for each control plane pod                                        |`1`|
+|`ControllerUID`                       | User ID for the control plane components                                             |`2103`|
+|`Identity.Issuer.ClockSkewAllowance`  | Amount of time to allow for clock skew within a Linkerd cluster                      |`20s`|
+|`Identity.Issuer.CrtExpiry`           | Expiration timestamp for the issuer certificate. It must be provided during install  | |
+|`Identity.Issuer.CrtExpiryAnnotation` | Annotation used to identity the issuer certificate expiration timestamp              |`linkerd.io/identity-issuer-expiry`|
+|`Identity.Issuer.IssuanceLifeTime`    | Amount of time for which the Identity issuer should certify identity                 |`86400s`|
+|`Identity.Issuer.TLS.CrtPEM`          | Issuer certificate. It must be provided during install                               |``|
+|`Identity.Issuer.TLS.KeyPEM`          | Key for the issuer certificate. It must be provided during install                   |``|
+|`Identity.TrustAnchorsPEM`            | Trust root certificate. It must be provided during install                           |``|
+|`Identity.TrustDomain`                | Trust domain used for identity                                                       |`cluster.local`|
+|`GrafanaImage`                        | Docker image for the grafana container                                               |`gcr.io/linkerd-io/grafana`|
+|`HeartbeatSchedule`                   | Config for the Heartbeat cronjob                                                     |`0 0 * * *`|
```

Comment:
Either use lower case for Heartbeat here, or use upper case for other components like identity, controller, tap etc.